contrib/ci/build_windows.sh

```diff
@@ -27,6 +28,8 @@ meson .. \
     -Ddocs=none \
     -Dhsi=false \
     -Dman=false \
+    -Dfish_completion=false \
```

Comment:
I'd actually argue we don't want bash completion either on Windows

Reply:
Agree, fixed.